src/components/forms/AddServiceTypeForm.js

```diff
@@ -1,38 +1,38 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { connect } from 'react-redux';
 import { Form, Input, Select, Modal, Menu, Dropdown } from 'antd';
-import { DownOutlined } from '@ant-design/icons';
-import CC_NumberInput from './CustomizableComponents/CC_NumberInput';
+// import { DownOutlined } from '@ant-design/icons';
+// import CC_NumberInput from './CustomizableComponents/CC_NumberInput';
```

Comment:
Good job spotting code that's not being used. Talking to both Ash and J will help narrow down if this will be needed later on, or should just be deleted.